# Generated for NaoQi method ALVisionVideoInput.getImgPosFromAngPos
# Returns normalized image position from camera angles in radians.

# camera angle values in radians.
float32[] angPos
---
# corresponding normalized position in the image [0.0 - 1.0]
float32[] value


